sys/auto_init/netif/auto_init_sx127x.c

```diff
@@ -21,16 +21,25 @@
 
 #include "log.h"
 #include "board.h"
+#ifdef MODULE_GNRC_LORAWAN
+#include "net/gnrc/netif/lorawan_base.h"
+#else
 #include "net/gnrc/netif/raw.h"
+#endif
 #include "net/gnrc.h"
 
 #include "sx127x.h"
 #include "sx127x_params.h"
 
+#if !defined(MODULE_GNRC_LORAWAN) || doxygen
 /**
  * @brief   Calculate the number of configured SX127x devices
  */
 #define SX127X_NUMOF        (sizeof(sx127x_params) / sizeof(sx127x_params_t))
+#else
+/* GNRC LoRaWAN only supports one interface so far */
+#define SX127X_NUMOF        (1)
```

Comment:
I wouldn't not do it this way: just keep the initial define but use an `assert(SX127X_NUMOF == 1)` if `MODULE_GNRC_LORAWAN` is defined, e.g. line 68.

Reply:
you are right. I will change it

Reply:
done!